core/src/main/java/nu/yona/server/subscriptions/service/UserService.java

```diff
@@ -114,6 +115,9 @@
 	@Autowired(required = false)
 	private WhiteListedNumberService whiteListedNumberService;
 
+	@Autowired(required = false)
+	private MigratePrivateUserDataService migratePrivateUserDataService;
```

Comment:
Please rename class and member into ``PrivateUserDataMigrationService``. That's a more common naming pattern.